web/templates/shared/form.html.eex

```diff
@@ -0,0 +1,19 @@
+<%= form_for @changeset, post_path(@conn, :create), fn f -> %>
```

Comment:
If its only abstracted in the rails app to be used in new and edit, but still in the context of posts, then maybe its better to just leave it under `web/templates/post` and not extract it to `web/templates/shared`?  Then we could probably get away with keeping the filename `form.html.eex`